Ladybird/AppKit/Application/ApplicationDelegate.mm

```diff
@@ -242,7 +254,7 @@ - (NSMenuItem*)createApplicationMenu
     auto* submenu = [[NSMenu alloc] initWithTitle:process_name];
 
     [submenu addItem:[[NSMenuItem alloc] initWithTitle:[NSString stringWithFormat:@"About %@", process_name]
-                                                action:@selector(orderFrontStandardAboutPanel:)
+                                                action:@selector(openAboutVersionPage:)
```

Comment:
Does about:about and this standard about dialog serve the same purpose? This dialog is meant to show things like the app logo, copyright info, etc. That does not seem to be what about:about is, and I'm not sure that an About menu item should navigate to it.

Reply:
A unified about page makes total sense. My point is that a page that displays a list of about: URLs isn't that.

Reply:
Oh! haha I'm more tired than I thought, ignore me! :)